Add dev-docs around Shared app sign on policies (#3062)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/apps/index.md

}
```

### Update application policy

> <ApiOperation method="put" url="/api/v1/apps/${applicationId}/policies/${policyId}" />

Reviewer: This is a `PUT` rather than a `POST`? Little unusual to have a `PUT` without a body so just wanted to confirm

Author: it is put , and the call is working without body, I can look into that more

Author: ![image](https://user-images.githubusercontent.com/84338599/155601301-9006afa4-99b6-4673-b895-8eeddf41fe62.png)

> Assign the application to the specific policy, it will un-assign the application form the currently assigned policy.

Reviewer 2: ```suggestion
Assign an application to a specific policy. This un-assigns the application from its currently assigned policy.
```
